complementing_a_strand_of_dna/reverse_complement_dna_string.py

```diff
@@ -0,0 +1,29 @@
+# take in a string of nucleobases and print the reverse complement
+import argparse as ap
+
+# store the complements of each base
+COMPLEMENTS = {
```

Comment:
i may consider making a module for this kind of stuff so that I don't have to keep adding it...we'll see how it goes in subsequent exercises, though I'm sure it will become quickly necessary